implementations/contracts/helpers/ConstantsChecker.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+// interfaces
+import {IERC725X} from "../interfaces/IERC725X.sol";
+import {IERC725Y} from "../interfaces/IERC725Y.sol";
+
+// constants
+import "../constants.sol";
+
+/**
+ * @dev Contract used to calculate interfacesId
+ */
+contract ConstantsChecker {
```

Comment:
I am not in favour of changing the name of this contract to from `ERC165InterfaceIDs` to `ConstantsChecker` as this contract is purely used for calculating ERC165 interface IDs only. The name `ConstantsChecker` implicitly translates to "not only ERC165 interface IDs but also any constants".

Also now the comment above the contract name makes it confusing. The contract name has the word "Constant" in it, but the comment states "Contract used to calculate interfacesId". I think it is misleading for the reader who reads this file for the first time.

Reply:
Reason explained above. Happy to revert the changes back :)